variables.tf

```diff
@@ -0,0 +1,90 @@
+variable "vpc_name" {
+  description = "(Required) The name of the VPC."
+}
+
+variable "vpc_cidr_block" {
+  description = "(Required) The CIDR block of the VPC."
+}
+
+variable "vpc_instance_tenancy" {
+  default     = "default"
+  description = "(Optional) Tenancy of instances spun up within the VPC (`default`, `dedicated`. Defaults to `default`)"
+}
+
+variable "region" {
+  description = "(Optional) The region in which to deploy the VPC."
```

Comment:
marked optional, but no default. Maybe grab the caller id's region?

Reply:
Fixed by removing the optional label.